package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "motrix-webextension",
+  "version": "2.0.0",
+  "description": "WebExtension for Motrix download manager",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --config=webpack.config.js --mode=production",
+    "start:firefox": "npm run build && web-ext run --firefox=firefoxdeveloperedition --source-dir ./dist/",
+    "start:edge": "npm run build && web-ext run -t chromium --chromium-binary edge.exe --source-dir ./dist/"
```

Comment:
chromium-based browsers not automatically starting. ( temporarily can load from developer mode. )